changelog.txt

```diff
@@ -1,7 +1,17 @@
-4.3.1
+4.3.2
 =============
+New Features:
+- **MSAL.NET now sends telemetry data to the /token endpoint in regards to the error code of the previous request, if applicable**. This will enable MSAL.NET to determine reliablity across public client application calls.
```

Comment:
This will enable the MSAL.NET team to compte reliablity across of public client application calls, and to proactively detect issues with the Microsoft identity platform (MSAL or the service) ?